src/devices/dresden_elektronik.ts

```diff
@@ -32,6 +32,13 @@ const definitions: Definition[] = [
         extend: extend.light_onoff_brightness(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['Hive'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
+        model: 'Hive', // Vendor model number, look on the device for a model number
```

Comment:
Is there any model number written on the device or product site? If yes use that here.

Reply:
no unluciily not. 
via Zigbee2MQTT i can read the following Data
'genBasic': {"modelId":"Hive","manufacturerName":"Phoscon"}
on the website is also no product code
https://phoscon.de/en/hive